mediathread/templates/projects/selection_assignment_edit.html

```diff
@@ -120,9 +120,15 @@ <h4 class="asset_title asset-{{item.primary.label}}">
                                     </a>
                                     </h4>
                                     <div class="asset-thumb">
-                                        <img class="gallery-item-thumb"
-                                         src="{{item.thumb_url}}"
-                                         alt="Item Thumbnail">
+                                        {% if item.thumb_url %}
+                                            <img class="gallery-item-thumb"
+                                             src="{{item.thumb_url}}"
+                                             alt="Item Thumbnail">
+                                        {% else %}{% ifequal item.media_type "image" %}
```

Comment:
fyi, i think you can do `{% if item.media_type == "image" %}` now. But this looks good to me.